rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RealPreference.java

```diff
@@ -62,10 +64,8 @@
   }
 
   @Override @NonNull public synchronized T get() {
-    if (!preferences.contains(key)) {
-      return defaultValue;
-    }
-    return adapter.get(key, preferences);
+    T value = adapter.get(key, preferences);
```

Comment:
This is too extreme, likely to break custom adapters that assume the old behavior.

Reply:
Oh, non-issue: `Adapter` is a private interface.